### ISSUE TITLE
term request: reservoir host

### ISSUE THREAD

**Issue by jhpoelen** [oborel/obo-relations#368]
from https://en.wikipedia.org/wiki/Host_%28biology%29 -

"Reservoir host - an organism that harbors a pathogen but suffers no ill effects. However, it serves as a source of infection to other species that are susceptible, with important implications for disease control. A single reservoir host may be reinfected several times.[14]"

Suggest to add as sub-term to "host of" http://purl.obolibrary.org/obo/RO_0002453 and "has host"  http://www.ontobee.org/ontology/RO?iri=http://purl.obolibrary.org/obo/RO_0002454 - 

![Screenshot from 2020-04-08 08-51-35](https://user-images.githubusercontent.com/1084872/78805490-51945980-7976-11ea-8c0b-b7777bec133e.png)

**Comment by wdduncan** [comment]
This issue is being closed because it has not had any activity in the last 2 years. If you think it is still worth pursuing, please reopen the issue and comment on its status.

**Comment by jhpoelen** [comment]
@wdduncan thanks for taking the time to manage open RO issues.

A reservoir host interaction is still very much needed. Please re-open this issue.

**Comment by wdduncan** [comment]
@jhpoelen I have reopened the issue. Do you have a plan for how to move this forward? Are you able to attend an RO meeting?

**Comment by jhpoelen** [comment]
@wdduncan thanks for taking the time to manage the open RO issues. The plan moving forward is to introduce the proposed term into RO. Would attending an RO meeting help achieve this?

**Comment by wdduncan** [comment]
@jhpoelen Yes, attending the RO meeting would be incredibly helpful! RO does not have funding and relies on volunteer efforts. Attending the meeting helps communicate the need to the rest of the RO community.

**Comment by jhpoelen** [comment]
@wdduncan thanks for your prompt reply - I'd be happy to attend an RO meeting. Where can I find the details?

**Comment by wdduncan** [comment]
The next meeting is on 2023-01-31 at 9am PT, 18.00 CET. Damion Dooley is charing. 

Minutes/agenda : http://bit.ly/ro-minutes
Zoom link: https://lbnl.zoom.us/j/97163538233?pwd=czlTWTRzb0YvY0sxa1hwWERZUTFEQT09

**Comment by jhpoelen** [comment]
@wdduncan thanks, I'll try to make an effort to join the meeting.

**Comment by wdduncan** [comment]
@jhpoelen I was not able to attend today's meeting. Was you term addressed?

**Comment by jhpoelen** [comment]
@wdduncan thanks for responding.

This term was not discussed this time! But I asked the organizer how to get the term on the calendar. Open to ideas . . .

**Comment by wdduncan** [comment]
I've added the NTR to the new term requests section of the [agenda for the next meeting](https://bit.ly/ro-minutes).

**Comment by jhpoelen** [comment]
@wdduncan As you probably noticed, I was unable to make it to the monthly RO meeting. I did have it on my calendar, and I was otherwise engaged. My apologies for setting the expectation to show up only to not show up. Please let me know if you were able to discuss the term, and if not, whether it is slated for discussion in the next meeting.

**Comment by wdduncan** [comment]
Hi @jhpoelen  
We did not get around to discussing the term. The next RO call is March 28, 2023 at 9am PT, 18:00 CET. It would be great if you could attend.
Here is the [Zoom link](https://lbnl.zoom.us/j/97163538233?pwd=czlTWTRzb0YvY0sxa1hwWERZUTFEQT09).

**Comment by jhpoelen** [comment]
@wdduncan thanks!

**Comment by wdduncan** [comment]
Thanks for joining the meeting @jhpoelen ! Sorry, we didn't have time to reach a resolution.  
BTW, have you looked at IDO [reservoir of infectious agent role](http://purl.obolibrary.org/obo/IDO_0000424)?

The next meeting is on 2023-04-25 (9am PT, 12pm ET, 6pm CET). I hope you can join.

**Comment by cmungall** [comment]
unfortunately I was unable to join the call. I agree with all @jhpoelen's comments.

RO has shortcut relations, it has always has shortcut relations. We have design patterns to relate the shortcuts to non-predicate representations. If there is a community willing to do the work in providing expert input on definitions and they fit in within existing structures, we should continue to include these.

**Comment by wdduncan** [comment]
> We have design patterns to relate the shortcuts to non-predicate representations.   

@cmungall I agree that we should include this relation, but which design patterns are you referring to? This [page](https://oborel.github.io/obo-relations/shortcut-relations/#shortcut-relations) doesn't provide much information.

Some shortcut relations have property chains defined for them. E.g., here is the property chain for [device utilizes material](http://purl.obolibrary.org/obo/RO_0017001):
<img width="450" alt="image" src="https://user-images.githubusercontent.com/3186638/228542923-6ab60ad0-a356-4b7f-8c0f-e19c27f56b7e.png">

However, I don't think all shortcut relations have property chains defined. 

@jhpoelen The relation needs a definition. Here is some info about definitions in RO:  
- https://oborel.github.io/obo-relations/annotation-properties/#definitions  
- https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/  
- https://philpapers.org/archive/SEPGFW.pdf

**Comment by jhpoelen** [comment]
@kephelps shared a reference with reservoir host definition:

Setting the Terms for Zoonotic Diseases: Effective Communication for Research, Conservation, and Public Policy https://www.mdpi.com/1999-4915/13/7/1356
